elasticsearch/client.go

```diff
@@ -4,11 +4,12 @@ import (
 	"encoding/json"
 	"math"
 	"time"
+	"context"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
 	"github.com/prometheus/common/model"
-	elastic "gopkg.in/olivere/elastic.v3"
+	"github.com/olivere/elastic"
```

Comment:
Could you update `client_test.go`? Since the pkg name changed, test case  has also been broken.

Reply:
Will do!

Reply:
Updated!